content/en/docs/ambient/install/multicluster/verify/index.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Verify the ambient installation
+description: Verify that Istio ambient mode has been installed properly on multiple clusters.
+weight: 50
+keywords: [kubernetes,multicluster]
+test: yes
+owner: istio/wg-environments-maintainers
+---
+Follow this guide to verify that your ambient multicluster Istio installation is working
+properly.
+
+Before proceeding, be sure to complete the steps under
+[before you begin](/docs/setup/install/multicluster/before-you-begin) as well as
+choosing and following one of the multicluster installation guides.
```

Comment:
some links missing here?

Reply:
One was incorrect and one was missing. Should be fixed, but I'll try and look through the Netlify preview